docs/_snippets/cloud/integrations/snowflake.mdx

```diff
@@ -32,3 +32,9 @@ CREATE NETWORK POLICY ELEMENTARY_CLOUD
 	BLOCKED_IP_LIST=()
 COMMENT='IP used by Elementary Cloud to sync the Elementary schema';
 ```
+
+If your company implemented other Account level network policies you can always activate the network policy for the specific user:
+
+```sql
+ALTER USER <user_name> SET NETWORK_POLICY = ELEMENTARY_CLOUD;
+```
```

Comment:
I'd write this as follows -
After creating a network policy you would need to activate the newly created network policy at either the account level or user level. 

To check if you already have a network policy activated at the account level you can run the following command - 
`SHOW PARAMETERS LIKE 'network_policy' IN ACCOUNT`

If you already have a network policy activated for your account, you can activate the network policy for the Elementary user as follows - 
`ALTER USER elementary SET NETWORK_POLICY = ELEMENTARY_CLOUD;`

If you want to activate it at the account level simply run - 
`ALTER ACCOUNT SET NETWORK_POLICY = ELEMENTARY_CLOUD;`

Reply:
Hmm they don't have to activate it. But on the other hand it won't hurt if they'll run both. So ok hh